### DISCUSSION TITLE
Migration from HA version to experimental

### DISCUSSION THREAD

**Discussion by Kratax** [mletenay/home-assistant-goodwe-inverter#179] [Q&A]
Hi, 
Thanks a lot for your work on these integration. I have a questions regarding moving from the official version to experimental while using docker installation of the home assistant... how to proceed to keep all the data? 
Is it ok to stop docker, copy the latest version to custom components and restart?
Thanks a lot!
Regards,
Lubos

**Comment by mletenay** [comment]
Just install HACS, then the "experimental" version from within and restart HA.
Data should be kept, even if you uninstall HACS version and go back to core.

**Reply by Puffo19** [reply]
Hello @mletenay,

I also want to switch from the HA version to experimental version. Therefore I deleted the HA version. In HACS I am not able to find your repository. 

The experimental checkbox in HACS is activated, see 
<img width="326" alt="2023-07-26 19_08_57-Einstellungen – Home Assistant" src="https://github.com/mletenay/home-assistant-goodwe-inverter/assets/115349946/db21a45b-1817-4ab1-8404-743b1b8cdf94">

But there is no match searching the HACS repository:
<img width="1362" alt="2023-07-26 19_15_36-Clipboard" src="https://github.com/mletenay/home-assistant-goodwe-inverter/assets/115349946/e3cc3217-3cb7-4865-b9cc-04a362eab638">

Trying to add the repository manually leads in following error message:
<img width="421" alt="2023-07-26 19_12_33-HACS – Home Assistant" src="https://github.com/mletenay/home-assistant-goodwe-inverter/assets/115349946/486ea5f3-0061-4224-9dfa-613ebe9ca411">

What made I wrong?

Thanks in advance an best regards,
Martin

**Comment by mletenay** [comment]
Just install the HACS add-on https://hacs.xyz/docs/setup/download and from there just search for goodwe

**Reply by Puffo19** [reply]
Just for your information. 
A new installation of HACS does not help. 
But activating the following option solved my problem:
<img width="323" alt="2023-07-27 19_32_05-Clipboard" src="https://github.com/mletenay/home-assistant-goodwe-inverter/assets/115349946/e11ac275-c49a-4000-8db3-2e5ece44f6c1">
